packages/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -121,10 +121,16 @@ class CellValueUpdater {
     }
 
     const { apiRef, getRowId } = this.options;
+    const cellParams = apiRef.current.getCellParams(rowId, field);
```

Comment:
From reading the existing code, it seems that `apiRef.current.isCellEditable` method is supposed to receive the result of `apiRef.current.getCellParams` method as an argument, so I decided to use `apiRef.current.getCellParams(...).isEditable` instead of `apiRef.current.isCellEditable` method.

Reply:
actually, after checking the implementation of the `isCellEditable` I see that it has some additional checks (also for the column definition), so it is still better to use it with the cell params that you are getting
https://github.com/mui/mui-x/blob/64fbda1d4694438ed2b176310b56115c5110368a/packages/x-data-grid/src/hooks/features/editing/useGridEditing.ts#L37-L54

In that case, you can ignore the comment on the removal of `!colDef.editable`